pages/common/ncmpcpp.md

```diff
@@ -0,0 +1,19 @@
+# ncmpcpp
+
+> A command line music player client for the Music Player Daemon.
+
+- Connect to a music player daemon on a given host and port:
+
+`ncmpcpp --host {{ip}} --port {{port}}`
+
+- Print the current song to stdout:
```

Comment:
What does it mean to print the song to stdout ? Does it show the lyrics or something else ?

Reply:
It shows the artist name and song name, I will clarify it when I'm at work.

Reply:
Ok, maybe better to say "Display metadata about the current song to console". Try to avoid tech jargon like "stdout"